### ISSUE TITLE
max sequence length=512 (as per BERT)?

### ISSUE THREAD

**Issue by vr25** [YangLinyi/HTML-Hierarchical-Transformer-based-Multi-task-Learning-for-Volatility-Prediction#2]
Hi,

What is the maximum sequence for the input text? I expected it to be 512 based on the assumption that you're using BERT. However, in the [paper](https://dl.acm.org/doi/pdf/10.1145/3366423.3380128), Table 1 describes 520.

Also, what are earning call transcript (text) statistics? Can you please describe the min, max, mean, and std. dev. values?

Thanks!

**Comment by YangLinyi** [comment]
Hi Vipula,

For your questions:
1) For the word-level transformer, we set a dynamic length by using the project "Bert-as-a-Service". Meanwhile, we set the max sequence length for the sentence-level transformer as 520 -- the same as the [ACL, 19] paper. Our max sequence length can be longer than 512 -- the default max length for BERT is 512 -- as we simplify the transformer in our customized architecture.

2) We used the earnings calls dataset published by https://github.com/GeminiLn/EarningsCall_Dataset that is publicly available. 

The statistic can be easily calculated on that.

Hope it can answer your queries.
